02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,60 @@
+let {people} = require('../data')
+
+const getPeople = (req,res)=>{
+    res.status(200).json({success:true,data:people })
+}
+
+const postPeople = (req,res)=>{
```

Comment:
Typically the convention is to name controller methods after the action being performed rather than the specific HTTP verb being used, so you would use "create" instead of "post", that way it's easier to tell what the function is actually doing. If using "post" in the name helps you keep track of it better though, doing it this way is totally fine for now. Also, this method only deals with a single person rather than multiple people so I would call it "postPerson" or "createPerson" rather than "postPeople".

Reply:
You are right, that makes more sense